people/YishayMendelsohn.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "ישי מנדלסון",
+  "github": "myishay",
+  "facebook": "https://www.facebook.com/myishay",
+  "linkedin": "https://www.linkedin.com/in/yishay-mendelsohn-174493133/",
+  "description": "מפתח בחברה datree.io ומנחה את הפודקאסט קוד פתוח יחד עם דימה ברוסילובסקי",
+  "website": "https://openkodpod.com/"
```

How about we use the FB community page in the meanwhile and land the PR?

```suggestion
  "website": "https://www.facebook.com/openkodpod"
```

---

@MichalPorag @lirantal It actually works now but tbh I also experienced some issues with is in the past two days, so I think it would be a good idea to use your suggestion for now @lirantal . Thanks!

---

Still not working for me. I think it might be related to some Netlify issues that are going on right now. Are you hosting it there?

---

No as far as I know... I guess it has something to do with the way we forward the address. But still, as you said, it would be better using the Facebook page for now.

---

@MichalPorag waiting for you to merge